### ISSUE TITLE
Have google scholar index zenodo entries

### ISSUE THREAD

**Issue by anandtrex** [zenodo/zenodo#1859]
I was wondering if there are any plans to have google scholar automatically index zenodo (software) entries and show citations like it does for normal research articles.

**Comment by amaatouq** [comment]
Pretty please!!! that's how researchers are evaluated these days ... I am missing a lot of citations on google scholar because of this!

**Comment by astrojuanlu** [comment]
Same here - somehow an old version of my package is indexed, but not any of the newer ones.

**Comment by caedmon5** [comment]
Agreed. We've just been discussing at Force11 how publication via Zenodo is becoming a norm for Working Groups. But they aren't indexed. Likewise, I use Zenodo as a personal repository in preference to ResearchGate and Academia.edu, but where Zenodo is a real repository, those others are indexed.

**Comment by ppanero** [comment]
Unfortunately, we are not in direct control over how Google Scholar indexes Zenodo pages. We are trying our best to follow specifications and best practices are provided by Google Scholar's documentation (and occasional communication with their team) and have seen mixed results.  

Since this issue has been raised multiple times by other users, if we have any news we will be posting about it publicly on Zenodo channels.
